src/components/header/Header.css

```diff
@@ -0,0 +1,19 @@
+.header{
+  width: 100vw;
+  height: 70px;
+  text-align: center;
+  font-size: 18px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgb(245, 244, 243);
+}
+
+.logo{
+  margin-right: 15px;
+  height: 50px;
+}
+.logo:hover{
+  transition: 0.2s;
```

Comment:
I don't think this will work because transition property is defined on the element itself and it requires at least two values the property on which you want to apply this and the duration